pygfx/renderers/wgpu/_renderer.py

```diff
@@ -354,6 +374,12 @@ def _set_wobject_pipelines(self):
             self.blend_mode, weakref.WeakKeyDictionary()
         )
 
+    def render_views(self, *views):
+        """Comvenience function to render the given views and then flush."""
+        for view in views:
+            self.render(view.scene, view.camera, viewport=view.viewport, flush=False)
```

Comment:
or... `view.renderer`? feels like this method belongs on view, not on renderer

Reply:
Hm... at the very least I think there should be a check that `view.renderer is self`